app/Locale/de_DE/translations.php

```diff
@@ -1184,44 +1184,44 @@
+    'Subtask removed on task #%d' => 'Teilaufgabe der Aufgabe #%d entfernt',
     'Hide tasks in this column in the dashboard' => 'Aufgaben in dieser Spalte im Dashboard ausblenden',
-    // '%s removed a comment on the task %s' => '',
-    // '%s removed a subtask for the task %s' => '',
-    // 'Comment removed' => '',
-    // 'Subtask removed' => '',
-    // '%s set a new internal link for the task #%d' => '',
-    // '%s removed an internal link for the task #%d' => '',
-    // 'A new internal link for the task #%d have been defined' => '',
-    // 'Internal link removed for the task #%d' => '',
-    // '%s set a new internal link for the task %s' => '',
-    // '%s removed an internal link for the task %s' => '',
-    // 'Automatically set the due date on task creation' => '',
-    // 'Move the task to another column when closed' => '',
-    // 'Move the task to another column when not moved during a given period' => '',
-    // 'Dashboard for %s' => '',
-    // 'Tasks overview for %s' => '',
-    // 'Subtasks overview for %s' => '',
-    // 'Projects overview for %s' => '',
-    // 'Activity stream for %s' => '',
-    // 'Calendar for %s' => '',
-    // 'Notifications for %s' => '',
-    // 'Subtasks export' => '',
-    // 'Tasks exportation' => '',
-    // 'Assign a color when the task is moved to a specific swimlane' => '',
-    // 'Assign a priority when the task is moved to a specific swimlane' => '',
-    // 'User unlocked successfully.' => '',
-    // 'Unable to unlock the user.' => '',
-    // 'Move a task to another swimlane' => '',
-    // 'Creator Name' => '',
-    // 'Time spent and estimated' => '',
-    // 'Move position' => '',
-    // 'Move task to another position on the board' => '',
-    // 'Insert before this task' => '',
-    // 'Insert after this task' => '',
-    // 'Unlock this user' => '',
+    '%s removed a comment on the task %s' => '%s hat einen Kommentar in der Aufgabe %s entfernt',
+    '%s removed a subtask for the task %s' => '%s hat eine Teilaufgabe in der Aufgabe %s entfernt',
+    'Comment removed' => 'Kommentar entfernt',
+    'Subtask removed' => 'Teilaufgabe entfernt',
+    '%s set a new internal link for the task #%d' => '%s hat eine neue interne Verbindung in der Aufgabe #%d erstellt',
+    '%s removed an internal link for the task #%d' => '%s hat eine interne Verbindung von der Aufgabe #%d entfernt',
+    'A new internal link for the task #%d have been defined' => 'Eine neue interne Verbindung für die Aufgabe #%d wurde definiert',
+    'Internal link removed for the task #%d' => 'Interne Verbindung in der Aufgabe #%d wurde entfernt',
+    '%s set a new internal link for the task %s' => '%s hat eine neue interne Verbindung in der Aufgabe %s erstellt',
+    '%s removed an internal link for the task %s' => '%s hat eine interne Verbindung von der Aufgabe %s entfernt',
+    'Automatically set the due date on task creation' => 'Ablaufdatum automatisch bei Erstellung einer Aufgabe setzen',
```

Comment:
Ablaufdatum has always been Ablaufdatum in the german translation, so I didn't want to change it now.